apitools/base/py/credentials_lib.py

```diff
@@ -13,7 +14,8 @@
 import oauth2client.locked_file
 import oauth2client.multistore_file
 import oauth2client.service_account
-import oauth2client.tools  # for flag declarations
+# It is needed for gflags declarations.
```

Comment:
nit: `It` is ambiguous (is it the next declaration or the previous one?). maybe either name the module or keep it on the same line?